zk/src/archive/web/js/zk/widget.js

```diff
@@ -1876,8 +1876,8 @@ wgt.$f().main.setTitle("foo");
 					if (!floating) this.setDomVisible_(node, visible);
 				} else if (visible) {
 					var zi;
-					if (floating)
-						this.setZIndex(zi = _topZIndex(this), {fire: true});
+					if (floating && !this.userZIndex)

```

Comment:
No need to add `&& !this.userZIndex` here.

In `setZIndex()`, if `opt.floatZIndex && this.userZIndex`, just don't do anything.

Reply:
ok